src/GraphQL/Execution/ExecutionOptions.cs

```diff
@@ -61,9 +61,6 @@ public class ExecutionOptions : IProvideUserContext
         /// <summary>Field and argument names are sanitized by the provided <see cref="INameConverter"/>; defaults to <see cref="CamelCaseNameConverter"/></summary>
         public INameConverter NameConverter { get; set; } = CamelCaseNameConverter.Instance;
 
-        /// <summary>Allows unhandled <see cref="Exception"/> stack traces to be serialized into GraphQL query result json along with exception messages; defaults to only <see cref="Exception.Message"/></summary>
-        public bool ExposeExceptions { get; set; }
```

Comment:
It would be difficult to support backwards compatibility, since the `DocumentExecuter` does not call the serializer, and `ErrorParser` does not receive a reference to `ExecutionResult`.